libpng find_package broken

**Describe the bug**
When installing libpng and attempting to use through cmake, I get the error in the log section below.

Package installed correctly and I followed the instructions it gave:
```
The package libpng:x64-linux provides CMake targets:

    find_package(libpng CONFIG REQUIRED)
    target_link_libraries(main PRIVATE png_static)
```

**Environment**
- OS: Ubuntu 18.04
- Compiler: gcc 7.4.0/cmake 3.14.3

**To Reproduce**
Steps to reproduce the behavior:
1. ./vcpkg install libpng
2. In my project's CMakeLists.txt do `find_package(libpng CONFIG REQUIRED)`

**Expected behavior**
I should be able to find libpng with `find_package`

**Failure logs** 
```
CMake Error at /home/sebastien/.ezvcpkg/2019.07/scripts/buildsystems/vcpkg.cmake:266 (_find_package):
  Could not find a package configuration file provided by "libpng" with any
  of the following names:

    libpngConfig.cmake
    libpng-config.cmake

  Add the installation prefix of "libpng" to CMAKE_PREFIX_PATH or set
  "libpng_DIR" to a directory containing one of the above files.  If "libpng"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Additional context**
vcpkg folder `installed/x64-linux/share/libpng` contains the following files:
```
libpng16.cmake
libpng16-debug.cmake
libpng16-release.cmake
```

Renaming to:
```
libpngConfig.cmake
libpngConfig-debug.cmake
libpngConfig-release.cmake
```

lets cmake find the package as expected.

--- a/ports/libpng/CONTROL
+++ b/ports/libpng/CONTROL
@@ -1,5 +1,5 @@
 Source: libpng
-Version: 1.6.37-2
+Version: 1.6.37-3
 Build-Depends: zlib
 Homepage: https://github.com/glennrp/libpng
 Description: libpng is a library implementing an interface for reading and writing PNG (Portable Network Graphics) format files.

--- a/ports/libpng/libpngConfig.cmake
+++ b/ports/libpng/libpngConfig.cmake
@@ -1,6 +1,6 @@
-# The upstream CMake exports its targets to a file named libpng.cmake
-# however, find_package(libpng CONFIG) doesn't work with that name.
+# The upstream CMakeLists.txt exports libpng's targets to a file named `libpng16.cmake`.
+# However, `find_package(libpng CONFIG)` expects a file named `libpngConfig.cmake` to exist instead.
 #
-# By includeing `libpng.cmake` form this file, find_package() will be 
-# able to find the exports `libpngConfig.cmake`.
-include(libpng16)
+# By including `libpng.cmake` from this file, `find_package(libpng CONFIG)` will work.
+get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
+include("${_DIR}/libpng16.cmake")

--- a/ports/libpng/libpngConfig.cmake
+++ b/ports/libpng/libpngConfig.cmake
@@ -1,0 +1,6 @@
+# The upstream CMake exports its targets to a file named libpng.cmake
+# however, find_package(libpng CONFIG) doesn't work with that name.
+#
+# By includeing `libpng.cmake` form this file, find_package() will be 
+# able to find the exports `libpngConfig.cmake`.
+include(libpng16)

--- a/ports/libpng/portfile.cmake
+++ b/ports/libpng/portfile.cmake
@@ -46,6 +46,7 @@ endif()
 
 vcpkg_fixup_cmake_targets(CONFIG_PATH lib/libpng)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/share/)
+file(COPY ${CMAKE_CURRENT_LIST_DIR}/libpngConfig.cmake DESTINATION ${CURRENT_PACKAGES_DIR}/share/libpng)
 
 file(COPY ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/libpng)
 file(RENAME ${CURRENT_PACKAGES_DIR}/share/libpng/LICENSE ${CURRENT_PACKAGES_DIR}/share/libpng/copyright)

